src/pages/DisplayFixtures.jsx

```diff
@@ -122,14 +112,16 @@ const DisplayFixtures = () => {
           },
           body: JSON.stringify(update),
         });
-        setAllPredictions([...predictions, prediction]);
-        setEditMode(false);
         // console.log(`Predictions updated in database: ${JSON.stringify(predictions)}`)
-
       } catch (error) {
         console.error('Failed to create/update listing:', error);
       };
     };
+    const fetchedPredictions = await getPredictions();
+    setAllPredictions(fetchedPredictions);
```

Comment:
since we have updated the DB through `PUT` API calls above, we can now be sure that DB has the most updated value. Hence, we can treat DB as our source of truth and `setAllPredictions` from fetched DB values. 

Remember by calling `setAllPredictions` and updating `allPredictions`, you'll trigger an effect in `App.jsx` (L151-L167) which would also update `userPredictions` and subsequently update all the views including `Prediction.jsx`

In summary, your source of truth flows like so:

DB --> `allPredictions` --> `userPredictions` --> `Prediction.jsx` fields